src/renderer/main/panels/configuration/Configuration.store.js

```diff
@@ -468,7 +468,9 @@ function create_configuration_manager() {
       const callback = () => {
         runtime.element_preset_load(x, y, element, preset).then(() => {
           const ui = get(user_input);
-          set(createConfigListFrom(ui));
+          let list = createConfigListFrom(ui);
+          list = afterUpdate(list);
```

Comment:
Sorry to be a pain in the ass, but `afterUpdate` could be confused with a svelte internal function! Please find a different name.

Otherwise thanks for the refactor, it seems better to me!

https://svelte.dev/docs/svelte#afterupdate